src/datetime/mod.rs

```diff
@@ -982,6 +962,30 @@ impl<Tz: TimeZone> Sub<Days> for DateTime<Tz> {
     }
 }
 
+impl<T, Tz: TimeZone> AddAssign<T> for DateTime<Tz>
+where
+    NaiveDateTime: Add<T, Output = NaiveDateTime>,
```

Comment:
This is not correct. The `Add` implementation of `Days` and `Months` with `DateTime` is timezone-aware. I think the correct way to do this is to just add the individual implementations.